src/flashbar/motion.scss

```diff
@@ -5,7 +5,7 @@
 
 @use '../internal/styles' as styles;
 @use '../internal/styles/tokens' as tokens;
-.flash-refresh {
+.flash-with-motion {
```

Comment:
Renaming because animated notifications now re not exclusive to Visual Refresh: also Classic gets animations if using the stacking feature.